scripts/django_tests_use_rusty_templates.patch

```diff
@@ -27,3 +27,163 @@ index ccfc327..1ab9ace 100755
+ 
+     @setup(
+         {
+@@ -197,34 +197,31 @@ class IncludeTagTests(SimpleTestCase):
+     def test_include_error07(self):
+         template = self.engine.get_template("include-error07")
+         with self.assertRaises(RuntimeError):
+-            template.render(Context())
++            template.render({})
+ 
+     @setup({"include-error08": '{% include "include-fail2" %}'}, include_fail_templates)
+     def test_include_error08(self):
+         template = self.engine.get_template("include-error08")
+         with self.assertRaises(TemplateSyntaxError):
+-            template.render(Context())
++            template.render({})
+ 
+     @setup({"include-error09": "{% include failed_include %}"}, include_fail_templates)
+     def test_include_error09(self):
+-        context = Context({"failed_include": "include-fail1"})
++        context = {"failed_include": "include-fail1"}
+         template = self.engine.get_template("include-error09")
+         with self.assertRaises(RuntimeError):
+             template.render(context)
+ 
+     @setup({"include-error10": "{% include failed_include %}"}, include_fail_templates)
+     def test_include_error10(self):
+-        context = Context({"failed_include": "include-fail2"})
++        context = {"failed_include": "include-fail2"}
+         template = self.engine.get_template("include-error10")
+         with self.assertRaises(TemplateSyntaxError):
+             template.render(context)
+ 
+     @setup({"include_empty": "{% include %}"})
+     def test_include_empty(self):
+-        msg = (
+-            "'include' tag takes at least one argument: the name of the "
+-            "template to be included."
+-        )
++        msg = "Expected an argument"
+         with self.assertRaisesMessage(TemplateSyntaxError, msg):
+             self.engine.get_template("include_empty")
+ 
+@@ -238,7 +235,7 @@ class IncludeTests(SimpleTestCase):
+         engine = Engine(app_dirs=True, debug=True)
+         template = engine.get_template("test_include_error.html")
+         with self.assertRaisesMessage(TemplateDoesNotExist, "missing.html"):
+-            template.render(Context())
++            template.render({})
+ 
+     def test_extends_include_missing_baseloader(self):
+         """
+@@ -249,7 +246,7 @@ class IncludeTests(SimpleTestCase):
+         engine = Engine(app_dirs=True, debug=True)
+         template = engine.get_template("test_extends_error.html")
+         with self.assertRaisesMessage(TemplateDoesNotExist, "missing.html"):
+-            template.render(Context())
++            template.render({})
+ 
+     def test_extends_include_missing_cachedloader(self):
+         engine = Engine(
+@@ -266,23 +263,21 @@ class IncludeTests(SimpleTestCase):
+ 
+         template = engine.get_template("test_extends_error.html")
+         with self.assertRaisesMessage(TemplateDoesNotExist, "missing.html"):
+-            template.render(Context())
++            template.render({})
```

Comment:
Should we gracefully handle if a Context object is passed instead of a dict ?

Reply:
Maybe, I'm not sure.